utils/build/docker/java/spring-boot-3-native/pom.xml

```diff
@@ -67,7 +67,7 @@
                 <artifactId>native-maven-plugin</artifactId>
                 <configuration>
                      <buildArgs>
-                        <buildArg>-J-javaagent:${agent.path}</buildArg>
+                        -J-javaagent:${agent.path} -J-Ddd.profiling.enabled=true --enable-monitoring=jfr
```

Comment:
Do we really want to activate profiling all the time, and could it impact other scenarios?